packages/peregrine/lib/store/actions/checkout/actions.js

```diff
@@ -1,7 +1,7 @@
 import { createActions } from 'redux-actions';
 
 const prefix = 'CHECKOUT';
-const actionTypes = ['BEGIN', 'EDIT', 'RESET'];
+const actionTypes = ['BEGIN', 'EDIT', 'RESET', 'SET_COUNTRIES'];
```

Comment:
Can you remove `GET_COUNTRIES` from the map in this file?

Reply:
Nice catch, thanks. Resolved in https://github.com/magento/pwa-studio/pull/1993/commits/68cd2b8ac59650c6a8b8a677a46d355adf1d1c0b.